src/helpers/index.ts

```diff
@@ -1,5 +1,11 @@
 import logger from 'decentraland-gatsby/dist/entities/Development/logger'
 
+export const CURRENCY_FORMAT_OPTIONS = {
```

Comment:
I think I moved this helper to the Grants helper file. If it's better here you can remove that from there and reuse it everywhere.